api/working-with-extensions/publishing-extension.md

```diff
@@ -93,6 +93,29 @@ Test your publisher's personal access token using [`vsce`](#vsce), while at the
 vsce login <publisher name>
 ```
 
+### Verify a publisher
+
+You can become a **verified publisher** by verifying ownership of an eligible domain associated with your brand or identity.
+
+Verifying your publisher will:
+* Help users confirm your publisher's authenticity.
+* Add a verified publisher badge to your extensions.
+* Give users quick access your brand's landing page or professional portfolio.
+
+![Verified publisher indicators in VS Code](images/verified-publishers/verified-publishers.png)
+
+Eligible domains meet the following criteria:
+
+* Cannot be a subdomain (i.e. <subdomain>.github.io, <subdomain>.contoso.com, or similar).
```

Comment:
We try to avoid Latin abbreviations.
Could this be "such as" or "for example"?

Reply:
It's probably clear enough if I just remove the "i.e." altogether. What do you think?

Reply:
Especially as you have "or similar"